mtgjson5/price_builder.py

```diff
@@ -113,6 +114,42 @@ def build_today_prices(self) -> Dict[str, Any]:
 
         return final_results
 
+    def build_today_prices_v2(self) -> MtgjsonPricesV2Container:
+        """
+        Build v2 price records by calling build_v2_prices() on each provider.
+
+        Aggregates v2 records from providers that support the v2 interface.
+        Providers without v2 support will return empty lists.
+
+        :return: Container with all v2 price records from today
+        """
+        if not self.all_printings_path.is_file():
+            LOGGER.error(
```

Comment:
I think this is wrong, we should download the AllPrintings if it doesn't exist for price builds